src/map/swipe.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <label>Swipe</label>
+  <input id="swipe" type="range" class="swipe-input" style="width: 100%;">
```

Comment:
style should not be added in the element directly, but in the CSS of the example instead